App.js

```diff
@@ -14,6 +14,8 @@ import {
   View,
   Text,
   StatusBar,
+  FlatList,
+  Dimensions
```

Comment:
This import is not being used. It's better to remove it. BTW, eslint will also alert you of this when trying to commit, it doesn't fix it but it alerts you.